playbooks/ansible-test-base/run.yaml

```diff
@@ -13,6 +13,14 @@
             file: '{{ _fetch.dest }}'
             name: galaxy_info
 
+    - name: Enable FIPS mode
+      when:
+        - ansible_distribution == "CentOS"
+        - hostvars[item].ansible_network_os in ['eos']
```

Comment:
This means all eos appliances will have FIPs enabled, this is a problem for paramiko. Since it won't support it.

We likely need an network_os check, maybe a denylist by default, along with libssh check